test_rclcpp/test/test_timeout_subscriber.cpp

```diff
@@ -40,8 +40,12 @@ TEST(CLASSNAME(test_timeout_subscriber, RMW_IMPLEMENTATION), timeout_subscriber)
   auto start = std::chrono::steady_clock::now();
 
   auto node = rclcpp::Node::make_shared("test_timeout_subscriber");
+
+  rmw_qos_policies_t custom_qos_policies = rmw_qos_policies_default;
```

Comment:
I think `rmw_qos_policies_default` should be passed as a default argument to `rclcpp::create_subscription` and `rclcpp::create_publisher`. This could go in a future PR because it's another major API change that would touch a lot of code (since the callback argument would have to go before the qos_policies argument).